### PR TITLE
add gitter link to README.md Closes #56

### LINKED ISSUE #56: Add link to gitter channel created in issue #55 to "README.md"

### PR DESCRIPTION
**Pull request by Aya-S**
Added link to gitter chat to README.md
PS: permission is needed to open the chat.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 
 
 This list is not yet complete—we'll keep adding more people.
+
+Gitter link: https://gitter.im/py-sprints/pandas-mentoring
```
Comment (datapythonista) [inline README.md]: Thanks @Aya-S 

I think the link won't be a link if it's not between `<` and `>`, like: `<link>`

Also, may be you can add couple of sentences on how to join it? It's also ok to do it in a follow up

Comment (Aya-S) [reply README.md]: Hey @datapythonista, 
I made the change on the local branch I created to add the link and pushed the changes to my forked repo. the changes automatically showed here, which should be Okay this time :D. can you confirm that the PR looks fine to you?

**Review by datapythonista — APPROVED** [review]
Ok, I guess github is "smarter", I think sphinx would show the url without a link.

Thanks!

**Comment by smiaa** [comment]
@datapythonista  I can see that **Outdated**"next to the README.md file above. Is this good to merge the changes from this PR or will it result in conflicts? At the same time, I see the messages that "This branch has no conflicts with the base branch"

**Comment by datapythonista** [comment]
The `Outdated` refers to the review comment. When someone comment on some changes, and then the line that the comment refers to changes, github assumes that the fix requested in the comment is applied.

In this case I commented in the line of the link to ask for the `<`/`>`, you added that, and github correctly assumed that you already implemented the change I proposed, and considers my comment outdated, which is correct.

**Comment by smiaa** [comment]
@datapythonista  Can I merge this request? But looks like I don't have write access?

**Comment by datapythonista** [comment]
@smiaa invited you

**Comment by smiaa** [comment]
@datapythonista, Thank you!

**Pull request merged**